daemon/tests/happy_path.rs

```diff
@@ -152,6 +156,40 @@ async fn collaboratively_close_an_open_cfd() {
     assert_next_state!(CfdState::Closed, maker, taker, order_id);
 }
 
+#[tokio::test]
+async fn force_close_an_open_cfd() {
+    let _guard = init_tracing();
+
+    let oracle_data = OliviaData::example_0();
+    let (mut maker, mut taker, order_id) =
+        start_from_open_cfd_state(oracle_data.announcement()).await;
+
+    // Taker initiates force-closing
+    taker.force_close(order_id).await;
+
+    deliver_event!(maker, taker, Event::CommitFinality(order_id));
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
+
+    // Delivering the wrong attestation does not move state to `PendingCet`
+    deliver_event!(maker, taker, dummy_wrong_attestation());
+
+    assert_next_state!(CfdState::OpenCommitted, maker, taker, order_id);
```

Comment:
I think this is not 100% sane because nothing would happen unless the CET timelock also expires.

Hence if you want to include this, I think we should deliver that event first. Then do

- wrong attestation
- sleep
- assert `OpenCommitted`
---
- correct attestation
- sleep
- assert `PendingCet`

Reply:
indeed, that is more foolproof. actioned the suggestion and added more comments in the code to highlight the behaviour.